src/routes/projects/ProjectReportingTab.svelte

```diff
@@ -215,9 +215,9 @@
                 bind:value={companyPerformanceMetricsResourceSize}
             />
         </div>
-        <div class="flex h-full flex-grow flex-row gap-4">
-            <div class="h-full w-full flex-1 overflow-x-hidden">
-                {#if companyCompletedItemsPerMonth === null}
+        <div class="flex min-h-[10rem] flex-grow flex-row gap-4">
```

Comment:
Nit: `min-h-[10rem]` is `min-h-40` equivalent

Reply:
Good call, looks like our tailwind version didn't support that, so I went ahead and updated it.